pkg/sql/colexec/external/external.go

```diff
@@ -256,6 +257,7 @@ func makeBatch(param *ExternalParam, plh *ParseLineHandler, mp *mpool.MPool) *ba
 		vec := vector.NewOriginal(typ)
 		vector.PreAlloc(vec, batchSize, batchSize, mp)
 		vec.SetOriginal(false)
```

Comment:
What is this going to do....

Reply:
just use vector.NewWithNspSize() instead of line 257 and 259 and 260.

Reply:
> 

fixed